ONBOARDING.md

```diff
@@ -1,4 +1,6 @@
-# Onboarding
+# Onboarding - DEPRECATED
+
+DEPRECATION WARNING: The latest version of GnosisVPN no longer requires manual onboarding - see [Usage](README.md#Usage) for details.
```

Comment:
This should be replaced with the current version of the onboarding, @esterlus @Jaguaras

Reply:
Onboarding is no longer required

Reply:
Then poof, goodbye file. 🍻